drivers/storage/portworx/component/csi.go

```diff
@@ -675,13 +678,13 @@ func getCSIDeploymentSpec(
 			deployment.Spec.Template.Spec.Containers,
 			v1.Container{
 				Name:            csiAttacherContainerName,
-				Image:           attacherImage,
+				Image:           "registry.k8s.io/sig-storage/csi-attacher:v4.3.0",
```

Comment:
This image should come from the csi_generator like the other csi images. Let's not hard code it here.

Reply:
Hold off on adding the attacher sidecar until I see why we need it. I am not convinced we need volume attachment objects and the volume attacher for windows support.

Reply:
We don't need the above sidecar for now.